swagger/src/main/scala/org/http4s/rho/swagger/SwaggerModelsBuilder.scala

```diff
@@ -0,0 +1,261 @@
+package org.http4s
+package rho
+package swagger
+
+import org.http4s.rho.bits.HeaderAST.HeaderRule
+import org.http4s.rho.bits.PathAST._
+import org.http4s.rho.bits.QueryAST.{QueryCapture, QueryRule}
+import org.http4s.rho.bits.ResponseGenerator.EmptyRe
+import org.http4s.rho.bits._
+
+import scala.reflect.runtime.universe.Type
+
+import scalaz._, Scalaz._
+
+class SwaggerModelsBuilder(formats: SwaggerFormats) {
```

Comment:
I think the whole class should be made private: I foresee little value
outside of our use.
On Feb 26, 2015 10:57 AM, "Carlos Encarnacion" <notifications@github.com>
wrote:

> In
> swagger/src/main/scala/org/http4s/rho/swagger/SwaggerModelsBuilder.scala
> <https://github.com/http4s/rho/pull/61#discussion_r25437693>:
>
> > @@ -0,0 +1,261 @@
> > +package org.http4s
> > +package rho
> > +package swagger
> > +
> > +import org.http4s.rho.bits.HeaderAST.HeaderRule
> > +import org.http4s.rho.bits.PathAST._
> > +import org.http4s.rho.bits.QueryAST.{QueryCapture, QueryRule}
> > +import org.http4s.rho.bits.ResponseGenerator.EmptyRe
> > +import org.http4s.rho.bits._
> > +
> > +import scala.reflect.runtime.universe.Type
> > +
> > +import scalaz._, Scalaz._
> > +
> > +class SwaggerModelsBuilder(formats: SwaggerFormats) {
>
> Where do you stand on this? The entire class should be private to the
> swagger package or which methods do you think we should expose?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/http4s/rho/pull/61/files#r25437693>.
>